api.swagger.yaml

```diff
@@ -9941,6 +9951,11 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/title-or-uuid-writable-list-item-schema'
+            regions:
+              description: "The user's regions. An array of region names or UUIDs."
+              type: array
+              items:
+                $ref: '#/components/schemas/title-or-uuid-writable-list-item-schema'
```

Comment:
I don't think we can use this schema for the writes as it would require a services API change for creating regions by name which is out of scope.

Reply:
done